assets/cart.js

```diff
@@ -42,13 +42,43 @@ class CartItems extends HTMLElement {
     }
   }
 
+  resetQuantityInput(id) {
```

Comment:
This is a similar function to what we have in `BulkAdd` in `global.js`. However, it doesnt make sense to extend that component yet since we dont have all the functionalities yet

When we do (this ticket https://github.com/orgs/Shopify/projects/6397/views/28?pane=issue&itemId=61489062) , let's do that